packages/arb-token-bridge-ui/src/components/common/SearchPanel/SearchPanelUtils.ts

```diff
@@ -1,6 +1,6 @@
 import { twMerge } from 'tailwind-merge'
 
-export const PanelWrapperClassnames = twMerge(
+export const panelWrapperClassnames = twMerge(
```

Comment:
Can we export a component instead? It feels weird to reuse this class everywhere

Reply:
i would love to, but wrapping `Popover.Panel` will be difficult as it gives `{({ close }) => {}` so this is much more straightforward